scripts/generate-test-data.sh

```diff
@@ -1,10 +1,16 @@
-set -e
+#!/usr/bin/env bash
+set -ex
 # Make the build environment image that is shared between multiple build targets
 ./scripts/make-build-env.sh
 
 # Setup the database on a named network
+db_version=$(<src/config/db_version)
 docker network create test-data
-docker run -d --rm --name db --network=test-data -p "8000:5432" docker.montagu.dide.ic.ac.uk:5000/montagu-db:master
+docker run -d --rm \
+  --name db \
+  --network=test-data \
+  -p "8000:5432" \
+  docker.montagu.dide.ic.ac.uk:5000/montagu-db:$db_version
```

Comment:
The only difference here is changing from hardcoded `master` to `$db_version`; I just also moved everything on to multiple lines.